### PR TITLE
Split out models into separate files

### LINKED ISSUE #130: Improve structure of models.py and schema.py
Currently, all of our GraphQL tables are defined in WePledge/backend/src/emissions/models.py

Django requires the tables to be defined in models.py but this is causing the file to be too big which makes looking for tables a bit complicated. 

An alternate way of storing the models is described [here](https://thewebdev.info/2022/04/02/how-to-split-models-py-into-several-files-with-python-django/#:~:text=with%20Python%20Django%3F-,To%20split%20models.py%20into%20several%20files%20with%20Python%20Django,root%20of%20the%20models%20folder.&text=We%20add%20__init__,class%20files%20inside%20app1%2Fmodels%20.)

Since this task will require moving around all the models. we should do a thorough smoke test of the application before merging

### PR DESCRIPTION
**Pull request by sami1riaz**

### CODE REVIEW

**Comment by sami1riaz** [comment]
I will create a separate PR for Schema.py as I don't want these PR's to get too large

File: backend/src/emissions/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" Django models for handling co2 emission data """
+
+from .customUser import CustomUser
+from .researchField import ResearchField
+from .institution import Institution
+from .workingGroup import WorkingGroup
+from .businessTrip import (BusinessTrip, BusinessTripGroup)
```
Comment (redfrexx) [inline backend/src/emissions/models/__init__.py:9]: I think the brackets are not necessary. Or are they?

File: backend/src/emissions/models/businessTrip.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
+
+from django.db import models
+from django.db.models import Sum
+
+
+from emissions.models import (CustomUser, WorkingGroup)
```
Comment (redfrexx) [inline backend/src/emissions/models/businessTrip.py:6]: same as above

**Review by redfrexx — APPROVED** [review]
Looks good!

File: backend/src/emissions/models/__init__.py (hunk 1 above)
Comment (sami1riaz) [reply backend/src/emissions/models/__init__.py:9]: They are not, I just couldn't resist lol

**Pull request merged**